src/tv2-common/helpers/graphics/timing.ts

```diff
@@ -68,46 +69,27 @@ export function FindInfiniteModeFromConfig(
 
 export function GetGraphicDuration(
 	config: TV2BlueprintConfig,
-	cue: CueDefinitionGraphic<GraphicInternalOrPilot>,
-	defaultTime: boolean
+	cue: CueDefinitionGraphic<GraphicInternalOrPilot>
 ): number | undefined {
 	if (config.showStyle.GFXTemplates) {
```

Comment:
Maybe invert the if, so you get something like this to avoid the nesting?
```
if (!config.showStyle.GFXTemplates) {
  return GetDefaultOut(config)
}
const template = findGFXTemplate(config, cue)
return template && template.OutType && !template.OutType.toString().match(/default/i)) 
  ? undefined
  : GetDefaultOut(config)
```